src/java.desktop/share/classes/java/awt/Robot.java

```diff
@@ -585,6 +631,16 @@ private static void checkScreenCaptureAllowed() {
         }
     }
 
+    /**
+     * Revokes the stored permission to capture screen data.
+     * Subsequent calls to {@link #getPixelColor(int, int)}
+     * and {@link #createScreenCapture(Rectangle)} may request
+     * a new permission from the user on applicable platforms.
+     */
+    public void revokeScreenCapturePermission() {
```

Comment:
Updated according to your comments.
I still left `revokeScreenCapturePermission` here for now, so that its documentation can be easily compared and aligned with the other documentation changes. 
After finalization it will be moved to #13803 (if we decide to keep the method)

Reply:
I agree with Phil (and Sergey, who mentioned the same earlier). I'm not sure I see enough motivation for this method to justify adding it. It seems more like a workaround for an implementation quirk of Wayland to me.

Reply:
Like I wrote elsewhere, I am not sure how the app would know when to call this.
And it is easy enough to use the preferences API to clear this out whilst we think more on this. BTW "reset" or "clear" might be better than "revoke" .. for all we know the current token denies access anyway and what we really mean is more like those other suggestions.